addons/website/views/snippets/s_numbers_boxed.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_numbers_boxed" name="Numbers boxed">
+    <section class="s_numbers_boxed pt80 pb80 o_cc o_cc5">
```

Comment:
Mobile layout it's not optimal, I propose to:
- increase the first column width
- enforce the first stat's font-size to `64px` by using inline style.

The last point it's a bit "borderline", I know... but there is not other way to get a very big title on mobile.
Anyway, enforcing an arbitrary font-size it's in the editor's possibilities  and the purpose of this text it's just decorative, so it's ok 👍 

| this PR | proposal |
|--------|--------|
| <img width="428" alt="image" src="https://github.com/user-attachments/assets/dc9d7410-4bf2-4c2a-b850-29b49b797a66"> | <img width="434" alt="image" src="https://github.com/user-attachments/assets/fbef2f83-defc-4255-9824-dbd3aa3c1645"> |